styles/styles.css

```
@@ -0,0 +1,696 @@
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.subscribe-container {
+  width: 544px;
+  margin: 0 auto;
+}
+
+.section-subscribe-images .subscribe-title,
+.section-subscribe-images .subscribe-desc {
+  color: #ffffff;
+}
+
+.subscribe-title {
+  margin-top: 0;
+  margin-bottom: 20px;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 36px;
+  text-transform: uppercase;
+  color: #000000;
+
+}
+
+.subscribe-desc {
+  margin-top: 0;
+  margin-bottom: 54px;
+  font-size: 22px;
+  line-height: 26px;
+  text-align: center;
+  color: #000000;
+}
+
+/* Footer */
+.footer-page  {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 45px 0;
+}
+
+.contacts-phone-link {
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 40px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #000000;
+  text-decoration: none;
+}
+
+/* Social list */
+.social-list{
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.social-list-item {
+  list-style-type: none;
+}
+
+.social-list-vk {
+  width: 24px;
+  height: 14px;
+  display: block;
+  mask: url("../images/svg/social/vk.svg") no-repeat center;
+  -webkit-mask: url("../images/svg/social/vk.svg") no-repeat center;
+  background-color: #83B3D3;
+  padding: 14px 12px;
+}
+
+.social-list-telegram {
+  width: 24px;
+  height: 14px;
+  display: block;
+  mask: url("../images/svg/social/telegram.svg") no-repeat center;
+  -webkit-mask: url("../images/svg/social/telegram.svg") no-repeat center;
+  background-color: #83B3D3;
+  padding: 14px 12px;
+}
+
+.social-list-youtube {
+  width: 24px;
+  height: 14px;
+  display: block;
+  mask: url("../images/svg/social/youtube.svg") no-repeat center;
+  -webkit-mask: url("../images/svg/social/youtube.svg") no-repeat center;
+  background-color: #83B3D3;
+  padding: 14px 12px;
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+/* Inner page */
+/* Filter */
+.filter-wrapper {
+  padding-top: 35px;
+  padding-right: 70px;
+  padding-bottom: 70px;
+  padding-left: 70px;
+  background-color: #E4F1F9;
+  background-image: url("../images/form/form-bg.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  color: #ffffff;
+}
+
+.filter-title {
+  font-weight: 700;
+  font-size: 60px;
+  line-height: 78px;
+  color: #ffffff;
+  margin-top: 0;
+  margin-bottom: 8px;
+}
+
+/* Breadcrumbs */
+.breadcrumbs-list {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+  margin-bottom: 40px;
+}
+
+.breadcrumbs-item {
+  margin-right: 28px;
+}
+
+.breadcrumbs-item:last-child {
+  margin-right: 0;
+}
+
+/* Housing */
+.page-wrap {
+    padding-top: 50px;
+    padding-bottom: 60px;
+}
+
+.housing-sort-title {
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 36px;
+  text-transform: uppercase;
+  color: #000000;
+  margin: 0;
+}
+
+.housing-sort-wrap {
+  width: 1060px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 40px;
+}
```

```suggestion
.housing-sort-wrap {
  width: 1060px;
  margin: 0 auto 40px;
  display: flex;
  justify-content: space-between;
  align-items: center;
}
```

Проверь все такие свойства